verilog/CST/package.cc

```diff
@@ -42,4 +42,22 @@ const verible::SyntaxTreeLeaf& GetPackageNameLeaf(const verible::Symbol& s) {
   return verible::GetSubtreeAsLeaf(s, NodeEnum::kPackageDeclaration, 2);
 }
 
+const verible::SyntaxTreeLeaf* GetPackageNameEndLabel(
+    const verible::Symbol& package_declaration) {
+  const auto* label_node = verible::GetSubtreeAsSymbol(
+      package_declaration, NodeEnum::kPackageDeclaration, 6);
+  if (label_node == nullptr) {
+    return nullptr;
+  }
+  const auto& package_name = verible::GetSubtreeAsLeaf(
+      verible::SymbolCastToNode(*label_node), NodeEnum::kLabel, 1);
+  return &package_name;
+}
+
+const verible::Symbol* GetPackageItemList(
+    const verible::Symbol& package_declaration) {
+  return verible::GetSubtreeAsSymbol(package_declaration,
+                                     NodeEnum::kPackageDeclaration, 4);
+}
```

Comment:
@fangism how can this function be tested? can you provide some similar tests?

Reply:
Example:

```systemverilog
package p;
  parameter int foo = 1;      // start matching tagged region here
  function void f();          // end matching here
endpackage
```

Follow the same outline as declaration_test.cc: analyze to get a syntax tree, match/find all package declarations, `GetPackageItemList` from each package (if not null), store the `TreeSearchMatch` into pas into `ExactMatchFindings`.

Does this help?